### DISCUSSION TITLE
How do you install plugins?

### DISCUSSION THREAD

**Discussion by AdamJSoftware** [opf/helm-charts#95] [Q&A] [answered]
I'm not entirely sure on how to install openproject community plugins with this helm chart. Any ideas?

**Comment by oliverguenther** [comment, marked as answer]
Hi @AdamJSoftware,

this would work in the same fashion as with the standard docker container. You need to provide your own container and override the registry here in the helm chart.

Here's a starting point on how to create the custom container: https://www.openproject.org/docs/installation-and-operations/installation/docker/#openproject-plugins
